test/test_language_server.py

```diff
@@ -72,12 +76,17 @@ def test_initialize(client_server):  # pylint: disable=redefined-outer-name
 @unix_only
```

Comment:
@andfoy, could we remove this decorator to this if things as expected on Windows? I mean, I think after PR #445 they should.

Reply:
The problem is that tests fail when we try to use multiprocessing on Windows

Reply:
ok, no problem then :+1: